app/assets/stylesheets/modules/filters.css.scss

```diff
@@ -41,3 +41,8 @@
   right: 0;
   top: 0;
 }
+
+.disabled-area {
+  pointer-events: none;
+  opacity: 0.4;
+}
```

Comment:
fyi @gitjul - that's what I added, feel free to adjust accordingly.

Reply:
looks ok, other disabled stuff had 0.4 opacity so it should be 👍